Source/ZoomNet/Resources/IMeetings.cs

```diff
@@ -74,6 +74,7 @@ public interface IMeetings
 		/// <param name="duration">Meeting duration (minutes).</param>
 		/// <param name="timeZone">The time zone for start time.</param>
 		/// <param name="password">Password to join the meeting. Password may only contain the following characters: [a-z A-Z 0-9 @ - _ *]. Max of 10 characters.</param>
+		/// <param name="default_password">Whether to generate a default password using the user's settings.</param>
```

Comment:
What happens if a developer specifies a value in the `password` parameter and also sets the `generatePassword` parameter to `true` (which sounds contradictory to me)? Does Zoom throw an exception? Or maybe one of these two values takes precedence over the other?

We should add a remark to inform developer what they should expect in this situation.

Reply:
To answer my own question: when you specify a custom password and you also set the new parameter to `true`, the custom password takes precedence and the Boolean parameter is ignored. As previously suggested, I'll throw an exception if developer specifies a custom password while at the same requesting a password to generated.